quipucords/api/common/util.py

```diff
@@ -112,6 +114,22 @@ def convert_to_boolean(value):
     return False
 
 
+def validate_query_param_bool(param, param_name='mask'):
+    """Validate that the query param is a boolean return the bool.
+
+    :param: param: The query param to evaluate
+    :return The value as a boolean or a validation error
+    """
+    if is_boolean(param):
+        return convert_to_boolean(param)
+    error = {
+        'mask': [_(
```

Comment:
I think you may want to replace `'mask'` here with `param_name`, additionally I think param_name needs to be added to the docstring.

Reply:
Done.